landing-page/content/common/concepts.md

```diff
@@ -0,0 +1,53 @@
+---
+url: concepts
+toc: true
+---
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+# Iceberg Concepts
```

Comment:
Do we want to call this "concepts"? That seems a bit abstract, though accurate. We might want to call it something more like "Overview" or "Walkthrough" so that people know to click and read through it. To me, "overview" is something that I'd read to get my bearings in a new project, while "concepts" sounds like it's going to be academic and not where I'd start out. If you ask me to go through a bunch of background before diving in, I'll probably go elsewhere. But if you show me a "here's what you need to know" page, I'm going there first. So it's all about how we present this.

Reply:
Good points, I went with "Overview" here and updated the PR.